### ISSUE TITLE
Multi-target classification in scikit-multilearn

### ISSUE THREAD

**Issue by bundit786** [scikit-multilearn/scikit-multilearn#66]
Hi developer team,
       I would like to know if scikit-multilearn can handle multi-target classification (multi-class problem for each label). I normally use Meka but it contains only a few of evaluation measures (lacking precision, recall,...) for multi-target setting. So I 'm looking for other alternatives, please advise.

Thank you.
Bundit B.

**Comment by ljvmiranda921** [comment]
Hi @bundit786! 

Correct me if I'm wrong but "multi-class problem for each label" sounds like Binary Relevance? In that case, it's already implemented here. 😄  

As for evaluation measures, usually what I do in my research is that I use [`sklearn.metrics`](http://scikit-learn.org/stable/modules/classes.html#sklearn-metrics-metrics) for classification. Some of them (like accuracy, hamming loss, f-score, precision/recall) can handle multi-label matrices for the predictions, `y_pred`, and ground-truth, `y_true`.

**Comment by bundit786** [comment]
Hi @ljvmiranda921 
    Thanks for your answer. According to the sklearn.metrics, can it handle multi-class data (within the multi-target learning)?
